api/buildstatus/buildstatus_controller.go

```diff
@@ -0,0 +1,73 @@
+package buildstatus
+
+import (
+	"net/http"
+
+	build_models "github.com/equinor/radix-api/api/buildstatus/models"
+	"github.com/equinor/radix-api/api/utils"
+	"github.com/equinor/radix-api/models"
+	"github.com/gorilla/mux"
+)
+
+const rootPath = "/applications/{appName}/environments/{env}"
+
+type buildStatusController struct {
+	*models.DefaultController
+	build_models.Status
+}
+
+// NewBuildStatusController Constructor
+func NewBuildStatusController(status build_models.Status) models.Controller {
+	return &buildStatusController{Status: status}
+}
+
+// GetRoutes List the supported routes of this handler
+func (bsc *buildStatusController) GetRoutes() models.Routes {
+	routes := models.Routes{
+		models.Route{
+			Path:                      rootPath + "/buildstatus",
+			Method:                    "GET",
+			HandlerFunc:               bsc.GetBuildStatus,
+			AllowUnauthenticatedUsers: true,
+		},
+	}
+
+	return routes
+}
+
+// GetBuildStatus reveals build status for selected environment
+func (bsc *buildStatusController) GetBuildStatus(accounts models.Accounts, w http.ResponseWriter, r *http.Request) {
+	// swagger:operation GET /applications/{appName}/buildstatus/{env} application getBuildStatus
```

Comment:
Should be `// swagger:operation GET /applications/{appName}/environments/{envName}/buildstatus environment getBuildStatus`